config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   root to: 'application#index'
+  match '*a', to: 'errors#routing_error', via: :all
```

Comment:
This route catchs all wrong urls and directs to the errors controller which returns not_found.
We did this because if not an exception is raised if someone hits a wrong url.
This made us run out of rollbar messages.